sys/Makefile.include

```diff
@@ -111,3 +111,7 @@ endif
 ifneq (,$(filter zptr,$(USEMODULE)))
   include $(RIOTBASE)/sys/zptr/Makefile.include
 endif
+
+ifneq (,$(filter dbgpin,$(USEMODULE)))
+  INCLUDES += -include $(RIOTBASE)/sys/include/dbgpin.h
+endif
```

Comment:
Also I'm not sure how clean this is (honestly don't have a clear opinion), but its the first occurrence of something like this. If we are OK with this handling for debug tools, I think the `trace` module could use it to, or the other way around, if `trace` requires manually including the header, then `dbgpin` should as well.

At the same time, if one is already modifying a file to add `dbgpin_cnt` calls, is it that much extra effort to include the header as well?

Reply:
pretty clean :-) 

>  I think the trace module could use it to

+1!

> At the same time, if one is already modifying a file to add dbgpin_cnt calls, is it that much extra effort to include the header as well?

IMHO not having this extra hassle is quite nice and makes those debug modules easier to use. Right now I even tend to prefer the 'always include an map to empty functions' option. When doing so, I was not able to measure a performance impact when compiling, so the costs seem to be negligible but we gain some flexibility when using the debug branch in different debug contexts...